.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": "explicit"
+  },
+  "eslint.format.enable": true,
```

Comment:
By enabling eslint as a formatter in vscode we can specify eslint as a formatter instead of prettier. Therefore even with the prettier plugin installed eslint autoformating on save will work.

This also allows us to use cmd+p -> format file without getting the error "eslint cannot format this file"